ci/build.sh

```diff
@@ -62,12 +62,6 @@ markStartOfBlock "Build the TestSuite"
     exit 1
   fi
 
-  Game/Scripts/BuildWorker.bat "TestSuiteServer" "Linux" "Development" "TestSuite.uproject" --skip-codegen
```

Comment:
There is a question if we want to build windows server here as an additional validation